packages/smart-contracts/src/lib/artifacts/index.ts

```diff
@@ -24,3 +24,7 @@ export * from './RequestDeployer';
  * ERC20 Transferable Receivable
  */
 export * from './ERC20TransferableReceivable';
+/**
+ * Admin
+ */
+export * from './Admin';
```

Comment:
Does it need to be exported?

Reply:
Yes we import it in another file to retrieve the safe address.
(It's not fully up to date yet, I'll make a PR when all the safes are ready)

Reply:
Thanks for clarifying. I had wondered if this list of Safe addresses was complete yet.